server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const path = require('path');
+const app = express();
+app.use(express.static(path.join(__dirname, 'build')));
+
+
+app.get('/api/cards', function (req, res) {
+ res.sendFile(path.join(__dirname, 'public', 'card-deck.json') )
+});
```

Comment:
here you are creating your own middleware (`function (req, res) ...`) to handle file download
I guess it's good you have it in place, because in the future you are going to serve cards from memory

but still I want to mention that you can use express.static to serve the json in the same way as you serve files from 'build' folder (line:5)

so for static card deck this could be simpler:
```
app.get('/api/cards', express.static(path.join(__dirname, 'public')));
```